apps/coscrad-frontend-e2e/src/e2e/tags/create-tag.e2e.cy.ts

```diff
@@ -0,0 +1,35 @@
+const commandLabel = `Create Tag`;
+
+describe(`Create Tag`, () => {
+    beforeEach(() => {
+        cy.visit('/');
+
+        cy.clearDatabase();
+
+        cy.executeCommandStream('users:create-admin');
+    });
+
+    describe(`when the user is logged in`, () => {
+        const firstTagLabel = 'plants';
+
+        beforeEach(() => {
+            cy.login();
+
+            cy.navigateToTagIndex();
+
+            cy.contains(commandLabel).as('commandButton');
+        });
+
+        describe(`when the form is complete`, () => {
+            it.only(`should succeed`, () => {
```

Comment:
What about
- when the form is incomplete
- when attempting to create a tag with the given label